doc/recipe/overview.rst

```diff
@@ -181,6 +181,11 @@ arguments):
 
 .. _Diagnostics:
 
+Preprocessor operations will be applied using the default order
+as listed in :ref:`preprocessor_functions`.
+Preprocessor tasks can be set to run in the order they are listed in the recipe
+by adding ``custom_order: true`` to the preprocessor definition.
```

Comment:
```suggestion
Preprocessor operations will be applied using the default order
as listed in :ref:`preprocessor_functions`.
Preprocessor tasks can be set to run in the order they are listed in the recipe
by adding ``custom_order: true`` to the preprocessor definition.

.. _Diagnostics:
```
It would be good to keep the label of the diagnostics section just above the section title